### ISSUE TITLE
Switch from tox to 

### ISSUE THREAD

**Issue by noisecapella** [mitodl/open-discussions#2292]
As a developer, I'd like to run tests the same way on xPRO and Open. Since we don't have multiple versions of python which need support and we are running tests inside a docker container, we can use just use pytest like we do on xPRO.

#### Designs and Mockups

See https://github.com/mitodl/mitxpro/pull/44. We should do something similar here.

#### Acceptance Criteria:

 - [ ] Travis runs the tests like it did before
 - [ ] Developers can run tests with `docker-compose run web pytest` instead of `docker-compose run web tox`
 - [ ] Environment variables in `.env` do not leak into the tests. This is important to prevent accidental S3 uploads or other similar things.
- [ ] Update docs to refer to pytest

#### Out of Scope

- Not going to do

**Comment by aliceriot** [comment]
will this make it so that each python related thing is under a different command? personally I am not a fan of the current setup where running `tox` does the linter, the formatter, etc before the tests run, and also prints out a whole lot of coverage-related stuff to the terminal. if we could have separate commands for each thing that would be :100: imo

**Comment by aliceriot** [comment]
related issue: #1564

**Comment by noisecapella** [comment]
Yes, separate commands. The general idea is to make it the same as what we do in xPRO.
